ui/app/common/patient-search/constants.js

```diff
@@ -5,8 +5,8 @@ Bahmni.Common.PatientSearch = Bahmni.Common.PatientSearch || {};
 Bahmni.Common.PatientSearch.Constants = {
     searchExtensionTileViewType: "tile",
     searchExtensionTabularViewType: "tabular",
-    tabularViewIgnoreHeadingsList: ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment"],
-    identifierHeading: ["ID", "Id", "id", "identifier", "DQ_COLUMN_TITLE_ACTION"],
+    tabularViewIgnoreHeadingsList: ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment", "extraIdentifier"],
+    identifierHeading: ["ID", "Id", "id", "identifier", "DQ_COLUMN_TITLE_ACTION", "Patient ID"],
```

Comment:
I believe both `identifierHeading` and `nameHeading` are arrays intended to determine key names. While I understand this isn't within the scope of this card, it would be beneficial to handle this. Instead of using constant arrays, configuring both `identifierHeading` and `nameHeading` would provide flexibility. Regarding `extraIdentifier`, this adjustment would grant implementers complete control over displayed content when modifying "emrapi.sqlSearch.activePatientsByLocation".

cc: @Arjun-Go, @angshu, @binduak, @gokultw

Reply:
@rahu1ramesh We will create a card to move this as a config for the later point of time

Reply:
@binduak @Arjun-Go It's essential to address the hardcoding of tabularViewIgnoreHeadingsList and identifierHeading, considering extraIdentifier and Patient ID is specific to the Cure implementation. This could lead to unexpected changes in either the product or CURE. Please prioritize resolving this to ensure that implementation-specific code isn't hardcoded.

Reply:
Change made as part of this commit - https://github.com/Bahmni/openmrs-module-bahmniapps/pull/628/commits/48f91045b673418e6dddaa51f33a7de6b972a10e